promgen/tests/test_models.py

```diff
@@ -34,3 +36,32 @@ def test_validators(self):
             validators.metricname(
                 "[[this.$el.ownerDocument.defaultView.alert(1337)]]",
             )
+
+    @mock.patch("django.dispatch.dispatcher.Signal.send")
+    def test_custom_save_rule(self, mock_post):
+        # Check if annotation["rule"] is automatically set to be {domain}/rule/{id} when creating a new rule
```

Comment:
```suggestion
        # Check if annotation["rule"] is automatically set to be {domain}/rule/{id} when creating a
        # new rule
```

Please wrap your lines at 100 characters, including comments.